src/tokens/theme.web.js

```diff
@@ -164,11 +166,11 @@ const darkTheme = {
       980: colors.white[980],
     },
     dark: {
-      730: colors.slate[930],
-      740: colors.slate[940],
+      930: colors.slate[930],
```

Comment:
this seems like breaking change. Just re-validate if this doesn't breaks anything in components. Because light theme has 730 and 740. At any point your themes can't be out of sync

Reply:
> this seems like breaking change. Just re-validate if this doesn't breaks anything in components. Because light theme has 730 and 740. At any point your themes can't be out of sync

Same way, Shade 500, 600 & 700 are added in dark theme while it's not there in light theme. 
I checked that there is no direct use of dark 730 and 740 in blade components.

Note: Whenever we will add dark & light themes in same project, we will need same aliases in both theme.